CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+[v0.13.0][v0.13.0_log]
+--------------
+
```

Comment:
### Added
- Enabled oe_verify_evidence() with a NULL format id to verify the legacy report generated by oe_get_report().
- Added the following SGX attestation claims from oe_verify_evidence():
     - OE_CLAIM_SGX_PF_GP_EXINFO_ENABLED
     - OE_CLAIM_SGX_ISV_EXTENDED_PRODUCT_ID
     - OE_CLAIM_SGX_IS_MODE64BIT
     - OE_CLAIM_SGX_HAS_PROVISION_KEY
     - OE_CLAIM_SGX_HAS_EINITTOKEN_KEY
     - OE_CLAIM_SGX_USES_KSS
     - OE_CLAIM_SGX_CONFIG_ID
     - OE_CLAIM_SGX_CONFIG_SVN
     - OE_CLAIM_SGX_ISV_FAMILY_ID
- Added the following fields for SGX KSS (Key Separation and Sharing) support:
     - FamilyID
     - ExtendedProductID